### ISSUE TITLE
Stylelint doesn't catch mistyped background-image value

### ISSUE THREAD

**Issue by Volker-E** [wikimedia/stylelint-config-wikimedia#121]
That's probably an upstream issue, but 
```
.link-https {
	background-image: images/external-ltr.svg );
```

isn't caught locally.

**Comment by edg2s** [comment]
Are you in LESS mode? I imagine that uses a less strict parser than the CSS parser. If there's a bug in the CSS parser it will be postcss-safe-parser.

**Comment by edg2s** [comment]
It looks like stylelint doesn't do much syntax validation beyond that implemented specific rules. None of this causes parser errors:
```css
.foo {
  background: foo.jpg );
  display: hello?;
  water: wet;
  width: wide;
}
```

You get `property-no-unknown` for `water`, but that's not a specific rule, not a parser error.

**Comment by edg2s** [comment]
Could experiment with something like https://github.com/csstree/stylelint-validator

**Comment by edg2s** [comment]
Also from #100:
> `background-image: ( 'images/tab-normal-fade.png' )` doesn't result in an error. Mind the missing `url` part.

**Comment by edg2s** [comment]
This regression caused by an invalid value: https://gerrit.wikimedia.org/r/c/mediawiki/core/+/800761

**Comment by edg2s** [comment]
In the above example, commas were added to an `animation` property where just spaces should be used:
`animation: rotate, 700ms, infinite, linear;`

While this is clearly an error, it is not invalid code, it just reads as 4 animations with short definitions:
```js
$0.style.animation='rotate, 700ms, infinite, linear'

console.log( $0.style.animation )
 '0s ease 0s 1 normal none running rotate, 700ms ease 0s 1 normal none running none, 0s ease 0s infinite normal none running none, 0s linear 0s 1 normal none running none'
```

**Comment by edg2s** [comment]
https://github.com/csstree/stylelint-validator has some support for LESS values but produces false positives on mixins. We could enable the rule for CSS only, but that would make it considerably less useful in MediaWiki where many (most?) files are LESS.
